src/components/Card.tsx

```diff
@@ -25,7 +25,9 @@ const imgStyle: ImageCSS = {
     clear: "both",
     display: "block",
     border: "0",
-    width: "100%"
+    width: "100%",
+    fontFamily: "Georgia, serif",
```

Comment:
Curious why we can't use Guardian fonts here?

Reply:
I will merge this for now, but I will try to look into custom font styling on the alt text, as I do remember hearing this in the past that custom fonts might not work on alt text.